### PR TITLE
Feat/opt cache

### PR DESCRIPTION
**Pull request by zachferland**
Based on https://github.com/3box/3box-pinning-server/pull/42, only merge after #42

### CODE REVIEW

**Comment by zachferland** [comment]
Resolves #35

**Comment by oed** [comment]
@zachferland Please connect to issue using the zenhub button instead of commenting! Do you have the plugin?
image

**Comment by zachferland** [comment]
yeah I can use zenhub instead, got the extension, just like how github handles it as well :) 

disable cache or cache service ? this was for disabling the redis cache for 'cache service'

**Comment by oed** [comment]
Yeah, use zenhub. It makes things much easier to track in the ZH board! 
Oh, I see I guess I misinterpreted the story. What do you think makes most sense of those two options?

**Comment by zachferland** [comment]
First the redis cache, since without out that, it makes running locally for development/testing easy and more clear. 

But now that you mention it, it will makes sense to allow the pining server to be run without the profile/cache service, so that someone can deploy a pinning node without it. I think I can roll it in to this story. Also in the future, some of the scaling solutions we mentioned will allow these two services to be separated anyways.

**Comment by oed** [comment]
Thanks @zachferland 🙏

**Comment by zachferland** [comment]
@oed mentioned this morning, but just just realized i didnt push up my recent changes, all up now

File: src/node.js [hunk 1]
```diff
@@ -14,13 +15,16 @@ const IPFS_PATH = process.env.IPFS_PATH
 const REDIS_PATH = process.env.REDIS_PATH
 
 const DAYS15 = 60 * 60 * 24 * 15 // 15 day ttl
+const runCacheService = argv.runCacheService !== 'false'
```
Comment (oed) [inline src/node.js]: so by default the string is `''` and `runCacheService` is true?

**Pull request merged**